provisioning/playbooks/couchbase/create_bucket.yml

```diff
@@ -0,0 +1,34 @@
+---
+- hosts: couchbase
+  sudo: False
+  gather_facts: False
+  tasks:
+    - name: Create cluster grouping
+      group_by: key={{ couchbase_server_node_role }}
+
+- hosts: primary
+  sudo: False
+  gather_facts: False
+  vars:
```

Comment:
These are variables specific for the bucket creation, not the entire group of machines.